components/desktop/KnitFooter.vue

```diff
@@ -1,13 +1,38 @@
 <template>
   <footer class="knit-footer">
-    Footer
+    <a href="/" class="knit-footer__link">
+      <img src="/knit-logotype.svg" class="knit-footer__logo" alt="Logo of Koło Naukowe IT">
+    </a>
+    <knit-navbar class="navbar--footy"/>
+    <span class="knit-footer__caption"> 2017 &copy; Koło Naukowe IT Politechnika Krakowska </span>
+    <div class="knit-footer__socialbox">
+      <ul class="knit-footer__socialbox-list">
+        <li class="knit-footer__socialbox-item">
+          <a href="/" class="knit-footer__socialbox-link">
+            <i class="flaticon-github-logo" aria-hidden="true"></i>
```

Comment:
Icons should not be wrapped in `<i>` tag. Use `span` instead. You can read here about it: https://github.com/w3c/html/issues/732

Reply:
Thanks, fixed :)